common/changes/@itwin/core-frontend/RealityDataAccessIntegrationTests_2022-01-10-20-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Mark url has resolved in internal flag in Raster Source for OSM Building entry key.",
```

Comment:
I don't know what this means. If it's not interesting to end users, change to empty string; otherwise clarify.

Reply:
Done, Changed to empty string